tests/www/api/experimental/test_endpoints.py

```diff
@@ -77,6 +140,24 @@ def test_task_info(self):
         self.assertNotIn('error', response.data.decode('utf-8'))
         self.assertEqual(200, response.status_code)
 
+        # Check for upstearm and downstearm task list
+        response = self.client.get(
+            url_template.format('example_bash_operator', 'run_after_loop')
+        )
+        self.assertNotIn('error', response.data.decode('utf-8'))
+        self.assertEqual(200, response.status_code)
+        response_content = json.loads(response.data.decode('utf-8'))
+
+        self.assertIn('"upstream_task_ids"', response.data.decode('utf-8'))
+        expected_upstream_task_list = ["runme_2", "runme_1", "runme_0"]
+        self.assertItemsEqual(sorted(response_content['upstream_task_ids']),
```

Comment:
This method does not exist in Python 3.7

Reply:
`assertListEqual`?